src/devices/sber.ts

```diff
@@ -53,7 +53,17 @@ const definitions: Definition[] = [
             battery({voltage: true, voltageReporting: true}),
         ],
     },
-    // Sber SBDV-00154 Smart leak sensor (fingerprint unknown)
+    {
+        fingerprint: [{modelID: 'TS0207', manufacturerName: '_TZ3000_c8bqthpo'}],
```

Comment:
Can this be merged with `TS0207_water_leak_detector`? (and add it as a `tuya.whitelabel` there)

Reply:
> This makes it harder to maintain as you have to search for a specific one in the sea of others

Why do you think this is the case? I want to prevent that we have the same code duplicated a lot of times for the same device (`TS0207`), otherwise we have to add every whitelabel as a separate defintion.

Reply:
Then I'd have to delete `sber.ts` and move everything into `tuya.ts`. There are a lot of branded Tuya devices that have their own vendor files as well. I understand that this is code duplication, but a necessary one for more coherent grouping by vendor. Also, since Sber got their Tuya license revoked, it is very much possible that they will release non Tuya based devices in the future. So grouping them in a vendor file seems more logical that splitting up.

Reply:
Then let's make an exception for this one. But I prefer to have whitelabels in `tuya.ts` to prevent a lot of code duplication.

Reply:
> Then let's make an exception for this one. But I prefer to have whitelabels in `tuya.ts` to prevent a lot of code duplication.

I'll keep this in mind, thanks!